python/ray/serve/schema.py

```diff
@@ -140,6 +141,12 @@ class Config:
             "Whether to enable access logs for each request. Default to True."
         ),
     )
+    additional_log_standard_attrs: List[str] = Field(
+        default_factory=list,
+        description=(
+            "Additional attributes to be added to all log records as standard attributes."
+        ),
+    )
```

Comment:
what is a "standard" attribute?

and what exactly are the entries to this list?

Reply:
standard here refer to standard python logging attributes